locales/en/translation.json

```diff
@@ -38,7 +39,9 @@
     "click-to-copy": "Click here to copy it",
     "snapshot-export-error": "Something went wrong trying to create that snapshot. Please try again.",
     "snapshot-import-error": "Something went wrong trying to load that snapshot. Please try again.",
-    "snapshot-not-found": "That snapshot doesn’t seem to exist. Check the link and try again."
+    "snapshot-not-found": "That snapshot doesn’t seem to exist. Check the link and try again.",
+    "classroom-export-complete": "Your project is ready to share to google classrooms",
```

Comment:
Let’s use “Google Classroom” (capitalized, singular) for the copy

Reply:
Can do!